cli/endpoints/online/triton/ensemble/create-endpoint-with-deployment-mir.yml

```diff
@@ -0,0 +1,26 @@
+name: mir-ensemble-model
+type: online
+auth_mode: aml_token
+traffic:
+    blue: 100
+deployments:
+    -   name: blue
+        model:
+           name: bidaf-ensemble
+           version: 1
+           local_path: ./models
+        code_configuration:
+            code:
+                local_path: ./code
+            scoring_script: dummy.py
+        environment:
+            name: triton-mir-endpoint
+            version: 1
+            docker:
+                image: mcr.microsoft.com/azureml/tritonserver-21.02-py38-inference:20210525.1
+        environment_variables:
+            AZUREML_EXTRA_REQUIREMENTS_TXT: requirements.txt
+        scale_settings:
+            scale_type: manual
+            instance_count: 1
+        instance_type: Standard_F8s_v2
```

Comment:
Any reason not to use  GPU SKU? Might as well highlight that Triton works well on GPUs right?

Reply:
Can you please share the available GPU sku for the workspace that runs the test.

Reply:
NCsv3